submissions/Akshata-4-2021-other/Akshata-4-2021-other.md

```diff
@@ -1 +1,49 @@
+## Opscientia Open Web Fellowship: A platform to store preprints and publications permanently 
+name: Akshata, email: makshata403@gmail.com, timezone: CEST, discordID: Akshata M#6695, current role: kernel fellow (KB4)
+
+### Overview
+
+The greatest treasure mankind can save for its future generations is knowledge. Knowledge in any form should be secured and never be lost. This project focuses on scientific knowledge of any form including research papers, preprints, writeups and thesis which could be open source. It will be built on 'Arweave' which assures that the papers uploaded on the platform will never be lost (would stay for thousands of years without any additional costs) and it is decentralized. 
+
+One other major goal of this project is that you can donate crypto tokens to the research paper, preprints, articles, thesis etc. which you find helpful. It is also possible to keep your papers public or private depending upon whether you want to safegaurd your work for yourself or if you like it to be open in public domain. The future goal would be to create a token called 'scholar' wich could be donated to important research projects asking for funds. As the platform will be built on Arweave, the necessary information and guidance about the token is already available.
+
+### Implementation Plan
```

Comment:
can you please expand on the implementation plan a little bit? i would list each phase of development, the resources you will require, and expected outcomes - doesn't have to be very detailed just a little more specific regarding what the workflow looks like so you can get good feedback